test/integration/rise-storage.html

```diff
@@ -354,25 +353,6 @@
           done();
         });
 
-        test("should retry storage request after rise-storage-api-error is encountered", function(done) {
```

Comment:
@rodrigopavezi As @donnapep just mentioned, RC is not running in these tests. Thanks